pkg/trtl/honu_test.go

```diff
@@ -116,6 +118,99 @@ func (s *trtlTestSuite) TestGet() {
 	require.True(proto.Equal(expectedMeta, reply.Meta))
 }
 
+// Test that we can call the Put RPC and get the correct response.
+func (s *trtlTestSuite) TestPut() {
+	require := s.Require()
+	alice := dbFixtures["alice"]
+	object := dbFixtures["object"]
+	ctx := context.Background()
+
+	// Start the gRPC client.
+	conn, err := s.connect()
+	require.NoError(err)
+	defer conn.Close()
+	client := pb.NewTrtlClient(conn)
+
+	// Put a value to a reserved namespace - should fail.
+	_, err = client.Put(ctx, &pb.PutRequest{
+		Namespace: "default",
+		Key:       []byte(object.Key),
+		Value:     []byte("foo"),
+	})
+	require.Error(err)
+
+	// Put a value without the key - should fail.
+	_, err = client.Put(ctx, &pb.PutRequest{
+		Namespace: object.Namespace,
+		Value:     []byte("foo"),
+	})
+	require.Error(err)
+
+	// Put without a value - should fail.
+	_, err = client.Put(ctx, &pb.PutRequest{
+		Namespace: object.Namespace,
+		Key:       []byte(object.Key),
+	})
+	require.Error(err)
+
+	// Put a value to the default namespace.
+	reply, err := client.Put(ctx, &pb.PutRequest{
+		Key:   []byte("testKey"),
+		Value: []byte("testVal"),
+	})
+	require.NoError(err)
+	require.True(reply.Success)
+	require.Empty((reply.Meta))
+
+	// Put a value to a valid namespace with return_meta=false.
+	reply, err = client.Put(ctx, &pb.PutRequest{
+		Namespace: alice.Namespace,
+		Key:       []byte(alice.Key),
+		Value:     []byte("arlo guthrie"),
+		Options: &pb.Options{
+			ReturnMeta: false,
+		},
+	})
+	require.NoError(err)
+	require.True(reply.Success)
+	require.Empty((reply.Meta))
+
+	// Put a value with return_meta=true.
+	expectedPID, err := strconv.Atoi((os.Getenv("TRTL_REPLICA_PID")))
+	require.NoError(err)
+	expectedRegion := os.Getenv("TRTL_REPLICA_REGION")
```

Comment:
Good solution to get the values that weren't hard coded, but I think this is a problem that I created -- sorry about that! I changed these lines: https://github.com/trisacrypto/directory/blob/main/pkg/trtl/trtl_test.go#L159-L162 not realizing that they were being used for the test. I will create a bug story to fix this by adding PID and region to the [test variables](https://github.com/trisacrypto/directory/blob/main/pkg/trtl/trtl_test.go#L28-L31) and making sure they're used in the [fixture](https://github.com/trisacrypto/directory/blob/main/pkg/trtl/trtl_test.go#L43-L53) and in the configuration. My bad!